src/predicates/in.jl

```diff
@@ -173,3 +173,25 @@ Base.in(p::Point, m::Multi) = any(g -> p ∈ g, parent(m))
 Tells whether or not the `point` is in the `domain`.
 """
 Base.in(p::Point, d::Domain) = any(e -> p ∈ e, d)
+
+
+"""
+    Base.in(p::Point{Dim}, splx::Simplex{K,Dim})
+
+Tells whether or not the point `p` is contained in the Simplex `splx`.
+Both must be embedded in the same dimension. Currently requires the simplex to have "full"
+parametric dimension, e.g. a triangle embedded in a plane.
+"""
```

Comment:
```suggestion
```

Reply:
Same reason as above. Methods of functions are not documented in this codebase because there are tons of methods, each of them doing the same thing with different geometries.

Reply:
> Both must be embedded in the same dimension. Currently requires the simplex to have "full"
parametric dimension, e.g. a triangle embedded in a plane.

I'd argue this comment is rather important, and it's it's not obvious from the other docstrings whether this would apply to them too (which imo means there's missing doc in the other methods).

Reply:
It sounds like an implementation detail that shouldn't reach end-users.

The way we think about docstring versus source code comments is the audience they target. The docstring targets the end-user that is simply calling `p in simplex` expecting the generic behavior of "point in simplex". The comments target the developer who must be aware of the limitation of the current implementation.

Writing good documentation is a challenge precisely because of that. We tend to be too technical in docstrings sometimes, but users just want to know if the function does the "right" thing.